server/questions.js

```diff
@@ -199,7 +199,7 @@ exports.checkAnswer = function(questionId, user, answer, callback) {
 		var value = (answer === question.answer);
 
 		if (userType === common.userTypes.ADMIN) {
-			return callback(null, value);
+			return callback(null, {correct: value, points: question.points});
```

Comment:
Do we want to return 0 points for the proof?

Reply:
It should not matter since we won't be using it for anything